bpfman/src/bpf.rs

```diff
@@ -616,22 +616,21 @@ impl BpfManager {
                             "bpfman-ns"
                         };
 
-                        let status = std::process::Command::new(bpfman_ns_path)
+                        let output = std::process::Command::new(bpfman_ns_path)
                             .args(prog_args)
-                            .status();
-
-                        debug!("bpfman-ns status: {:?}", status);
-
-                        match status {
-                            Ok(s) => {
-                                if !s.success() {
-                                    debug!("bpfman-ns returned non-successful result: {:?}", s);
+                            .output();
+
+                        match output {
+                            Ok(o) => {
+                                if !o.status.success() {
+                                    debug!(
```

Comment:
Shouldn't this just be `info!`? If there's an error we should always know about it